src/scss/sections/_testimonials-carousel.scss

```diff
@@ -0,0 +1,3 @@
+.testimonials-carousel {
+  background-color: $gray-100;
```

Comment:
please let's use bg class for this, if there is not, let's create it in utilities